README.md

```
@@ -111,48 +93,72 @@ cd zerostarter
+📖 **[Deployment guides →](https://zerostarter.dev/docs/deployment/vercel)**
+
+## 🤝 Contributing
 
-📖 **[View contributing guidelines →](https://zerostarter.dev/docs/contributing)**
+Contributions are welcome. Please read the [contributing guide](https://zerostarter.dev/docs/contributing) first.
 
-## ❤️ Amazing Contributors
+## ❤️ Contributors
 
 <a href="https://github.com/nrjdalal/zerostarter/graphs/contributors">
   <img src="https://contrib.rocks/image?repo=nrjdalal/zerostarter" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)